__tests__/sum.test.js

```diff
@@ -0,0 +1,4 @@
+import { shallow } from 'enzyme';
```

Comment:
I encourage you to give it a try to use React Testing Library [instead of Enzyme's `shallow` render](https://kentcdodds.com/blog/why-i-never-use-shallow-rendering).